scripts/build.sh

```diff
@@ -522,24 +514,6 @@ case "$product-$platform-$method" in
         build \
         test
       fi
-
-    pod_gen FirebaseStorage.podspec --platforms=macos --clean
-    RunXcodebuild \
-      -workspace 'gen/FirebaseStorage/FirebaseStorage.xcworkspace' \
-      -scheme "FirebaseStorage-Unit-unit" \
-      "${macos_flags[@]}" \
-      "${xcb_flags[@]}" \
-      build \
-      test
-
-    pod_gen FirebaseStorage.podspec --platforms=tvos --clean
```

Comment:
Just so I understand better, why are we removing these from the build script? Are your changes to the `pod-lib-lint` task in `storage.yml` effectively replacing these parts of the build script?

Reply:
Yes. The unit tests now get run from `pod lib lint`. It's just recently possible because CocoaPods 1.10 introduced the feature of being able to select test_spec's to run instead of the all or none that it was previously.